ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x__4.13.yaml

```diff
@@ -43,6 +43,31 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: acceptance-lp-interop
+  cron: 0 6 * * 1
+  remote_api: true
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+    test:
+    - as: sbo-acceptance-tests
+      cli: latest
+      commands: |
+        export CATSRC_NAME=redhat-operators
+        export OPERATOR_CHANNEL=stable
+        export OPERATOR_PACKAGE=rh-service-binding-operator
+        export SKIP_REGISTRY_LOGIN=true
+        ./install.sh
+        export TEST_ACCEPTANCE_START_SBO=remote
+        export EXTRA_BEHAVE_ARGS="--tags=~@crdv1beta1 --tags=~@olm-descriptors --tags=~@upgrade-with-olm --tags=~@disable-openshift-4.13 --tags=~@disable-openshift-4.12 --tags=~@disable-openshift-4.8+ --tags=~@disable-openshift-4.9+ --tags=~@disable-openshift-4.10+ --tags=~@disable-openshift-4.11+ --tags=~@disable-openshift-4.12+ --tags=~@disable-openshift-4.13+"
+        make -k VERBOSE=2 -o registry-login test-acceptance test-acceptance-artifacts
+        rename TESTS junit_TESTS /logs/artifacts/acceptance-tests/TESTS*.xml
+      from: src
+      resources:
```

Comment:
do you need to add limits to cpu/memory

Reply:
This was recommended by the contact that I had first worked with, I havn't proven whether or not it is needed, though this implementation hasn't led to any problems.